test/helpers/practical_supports_helper_test.rb

```diff
@@ -0,0 +1,103 @@
+require 'test_helper'
+
+class PracticalSupportsHelperTest < ActionView::TestCase
+  describe 'practical support options' do
+    describe 'with a config' do
+      before { create_practical_support_config }
+
+      it 'should include the option set' do
+        expected = [
+           nil,
+          "Metallica Tickets",
+          "Clothing",
+          ["Travel to the region", "Travel to area"],
+          ["Travel inside the region", "Travel inside area"],
+          ["Lodging", "Lodging"],
+          ["Companion", "Companion"]
+        ]
+        assert_same_elements expected, practical_support_options
+      end
+    end
+
+    describe 'without a config' do
+      it 'should create a config and return proper options' do
+        assert_difference 'Config.count', 1 do
+          @options = practical_support_options
+        end
+
+        expected = [
+          nil,
+          ["Travel to the region", "Travel to area"],
+          ["Travel inside the region", "Travel inside area"],
+          ["Lodging", "Lodging"],
+          ["Companion", "Companion"]
+        ]
+        assert_same_elements expected, @options
+        assert Config.find_by(config_key: 'practical_support')
+      end
+    end
+
+    describe 'with an orphaned value' do
+      it 'should push onto the end' do
+        expected = [
+          nil,
+          ["Travel to the region", "Travel to area"],
+          ["Travel inside the region", "Travel inside area"],
+          ["Lodging", "Lodging"],
+          ["Companion", "Companion"],
+          'bandmates'
+        ]
+        assert_same_elements expected, practical_support_options('bandmates')
+      end
+    end
+  end
+
+  describe 'practical support source options' do
+    describe 'with external pledge source config' do
+      before { create_external_pledge_source_config }
+
+      it 'should include the option set' do
+        expected = [
+          nil,
+          'DC Abortion Fund',
+          'Baltimore Abortion Fund',
+          'Tiller Fund (NNAF)',
+          'NYAAF (New York)',
+          'Clinic',
+          'Other (see notes)'
+        ]
+        assert_same_elements expected, practical_support_source_options
+      end
+    end
+
+    describe 'without a config' do
+      it 'should create a config and return options' do
+        assert_difference 'Config.count', 1 do
+          @options = practical_support_source_options
+        end
+
+        expected = [
+          nil,
+          'DC Abortion Fund',
+          'Clinic',
+          'Other (see notes)'
+        ]
+        assert_same_elements expected, @options
+        assert Config.find_by(config_key: 'external_pledge_source')
+      end
+    end
+
+    describe 'with an orphaned value' do
+      it 'should push orphaned value onto the end' do
```

Comment:
It looks like this will only support the orphaned value on the one pulldown it was selected on, not all of them. This seems okay-ish to me, but just wanted to make sure we're explicitly okay with that.

Reply:
I don't think I know what you mean by this, can you clarify?

Reply:
If there are multiple pulldowns and one has an orphaned value, that orphaned value will be an option on only one of the pulldowns. Anyways, not a big deal either way.